.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+#### Issue Summary
+
+A summary of the issue and the environment in which it occurs. If suitable, include the steps required to reproduce the bug. Please feel free to include screenshots, screencasts, code examples.
+
+
+#### Steps to Reproduce
+
+1. This is the first step
+2. This is the second step
+3. Further steps, etc.
+
+Any other information you want to share that is relevant to the issue being reported. Especially, why do you consider this to be a bug? What do you expect to happen instead?
+
+#### Technical details:
+
+* sendgrid-python Version: master (latest commit: [commit number])
```

Comment:
Please change this line to say `smtpapi-python` instead of `sendgrid-python`

Reply:
@mbernier now u can have a look